test/Tests.Infrastructure/NightlyBuildAzureStorageEmulatorFact.cs

```diff
@@ -7,7 +8,7 @@ public class NightlyBuildAzureStorageEmulatorFact : AzureStorageEmulatorFact
         public NightlyBuildAzureStorageEmulatorFact([CallerMemberName] string memberName = "")
             : base(memberName)
         {
-            if (NightlyBuildTheoryAttribute.IsNightlyBuild == false)
+            if (RavenTestHelper.IsRunningOnCI && NightlyBuildTheoryAttribute.IsNightlyBuild == false)
```

Comment:
we do not want this. If you need to force running nightly build tests then change `NightlyBuildTheoryAttribute.Force` to `true`

Reply:
done